dlup/data/transforms.py

```diff
@@ -80,19 +80,29 @@ def convert_annotations(
         if not (curr_annotation.label in index_map):
             continue
 
+        original_values = None
+        interiors = [np.asarray(pi.coords).round().astype(np.int32) for pi in curr_annotation.interiors]
+        if interiors is not []:
+            original_values = mask.copy()
+            holes_mask = np.zeros(region_size, dtype=np.int32)
+            # Get a mask where the holes are
+            cv2.fillPoly(holes_mask, interiors, 1)
+
```

Comment:
Can this be explained/described somewhere? Either in comments or docstring? I don't know what this adds to the function now.